README.md

```diff
@@ -163,594 +104,281 @@ The invoker MUST be the UCAN delegator. Their DID MUST be authenticated in the `
+|-------|------------|----------|-----------------------------------------------------------|
+| `iss` | `DID`      | Yes      | The DID of the [Invoker]                                  |
+| `aud` | `DID`      | Yes      | The DID of the intended [Executor]                        |
+| `run` | `&Task`    | Yes      | The enclosed [Task]'s CID                                 |
+| `cau` | `&Receipt` | No       | An OPTIONAL CID of the [Receipt] that enqueued the [Task] |
+
+### 3.3.2 Invocation (Envelope)
+ 
+| Field | Type                 | Required | Description                                              |
+|-------|----------------------|----------|----------------------------------------------------------|
+| `uci` | `&InvocationPayload` | Yes      | The CID of the [Invocation Payload]                      |
+| `sig` | `&Signature`         | Yes      | A signature by the Payload's `iss` over the CID in `uci` |
 
-## 5.3 DAG-JSON Example
+## 3.4 Examples
 
-### 5.3.1 Single Invocation
+### 3.4.1 Interacting with an HTTP API
 
-```json
+```js
 {
-  "bafy...createBlogPost": {
-    "on": "https://example.com/blog/posts",
-    "call": "crud/create",
-    "input": {
-      "headers": {
-        "content-type": "application/json"
-      },
-      "payload": {
-        "title": "How UCAN Tasks Changed My Life",
-        "body": "This is the story of how one spec changed everything...",
-        "topics": [
-          "authz",
-          "journal"
-        ],
-        "draft": true
-      }
-    }
-  },
-  "bafy...auth": {
-    "scope": [
-      {
-        "/": "bafy...createBlogPost"
-      }
-    ],
-    "s": {
-      "/": {
-        "bytes": "7aEDQPPhXNvtVb5/T+O40xXU6TSgJZDFnlVaV3GMlaEo/dvxtyaCLm8uUsFK4xzQsQd82QQUYA6fK506XqjghRlucAQ"
-      }
-    }
-  },
-  "bafy...invocation": {
-    "v": "0.1.0",
-    "run": {
-      "/": "bafy...createBlogPost"
-    },
-    "auth": {
-      "/": "bafy...auth"
-    },
-    "prf": [
-      {
-        "/": "bafy...ucanProof"
-      }
-    ]
-  }
+  "sig": {"/": {bytes: "7aEDQIscUKVuAIB2Yj6jdX5ru9OcnQLxLutvHPjeMD3pbtHIoErFpo7OoC79Oe2ShgQMLbo2e6dvHh9scqHKEOmieA0"}},
+  "inv": {                                                                    //           ┐
+    "iss": "did:plc:ewvi7nxzyoun6zhxrhs64oiz",                                //           │
+    "aud": "did:key:z6MkrZ1r5XBFZjBU34qyD8fueMbMRkKw17BZaq2ivKFjnz2z",        //           │
+    "run": cid({                                                              //  ┐        │
+      "act": cid({                                 // ┐                           │        │
+        "nnc": "&NCC-1701-D*",                     // │                           │        │
+        "cmd": "crud/create",                      // │                           │        │
+        "arg": {                                   // │                           │        │
+          "uri": "https://example.com/blog/posts", // │                           │        │
```

Comment:
obviously, these args are different for different invocations. Can we standardize on a URI field that is Option<Uri> for any arg. It seems important enough that it can't just be treated as an any? I could be wrong, but it seems to be important generally.